src/components/Layout.tsx

```diff
@@ -20,7 +22,12 @@ export const Layout = ({ children }: PropsWithChildren) => {
     } 
   };
 
+  const get_reward_tiers = async () => {
+    set_tiers(await get_all_reward_tiers());
+  }
+
   useEffect(() => { do_login_check() }, []);
+  useEffect(() => { get_reward_tiers() }, []);
```

Comment:
we shouldn't have 2 use effects. just put it after the `do_login_check()`

Reply:
ah I was following what we did in another file and didn't know for sure if we were able to double up like that